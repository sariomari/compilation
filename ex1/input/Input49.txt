]	LFRquXbFnilarray[81
:=extends-/**/array}8460
{;//]16
=;600
"".array62
-classint(<48
{,-carrayvoid({string;517
void}/*?!]);?*/<("l"]class"zGSzAbLffRC"2749
310
(classvoidvoidvoid(,++>:=classz,,	//I>)+array}string33
,40
(/*!!l
](2
E;*/{ /*3
D?;X!!!Y;)?;f)
*/4000
(arrayvoid//?(?o
500
6
},stringclass9837
nil[}>""6562
string{5
:=voidnil[{Cf,9539
7888
+{=//}!!?];
Ad}{8
24
extends+]910
777
:=class.void.;:=]//!;).?
<//
{//3!)void}-;
"f")(d:=string}52
-{extends=extends
a2lEextendsarray<"zNb"nil5699
//}jM;
(5886
=<+];class)39
,9
int
<extends-{<extends};7671
)0
]3
178
=41
{,string)57
classarray(>string+-void=0

])extendsvoid,8020
(int.<arraystring<[:=-/*
!z*//*6;?*/stringvoid721
void)-nil}nilvoid]+{M62>void]intvoidint<<9722
xB+(,void)]//(;}X?))}]!!]})]F}]bL
class<4047
;]}intarray//{};H)3]](](]]]
string//(?783(b)}X9V?;}?G?N])WL};J
:=.}.]-//P>0
87
68
5
/*K]B]!)}*/ /*];?L;s
(?!;p(DL*/extendsint/*
]j}khQ]!*/<;nil/*!1(?!]q!;*/5
yR7sz40x2ETc/*
c;}
;!9b}8])S?)
!?](v)a*/nil=6
jDKVQBBtclass-intstringstring<rcTextends=nil(/*D)]*/279
nil}="".	{class-	nil""<,+extends;stringvoid<5209
U(781
914
array,//;	4
;class//(/}
+class+class+[]int{:=<-[}58
>[>]{)string/*v]M])7}
}]!;B}
*/int;()intstring,void[ =	string=={]intlr:=73
extends=,int	class{class78
		[class"A"/*g(;}Z*/84
(]./*!1Ox;H2]
}*/=3

+nil+int
:={classextends,1556
({extends,);nil( classstring=hint,83
526
=];intint>,array.>[9284
[/*
}V}!

;
?[]A
}]]
kp1
9K?8}]}!
]S)}?)];vF(*/<=>{ extends(4
<:=arrayextends5
classyclass)711
}extends/*7*/class=extends//Z
stringy1/*;?*/(([/*!}?XVk;;?;;
(}
WS80b]
0?3]8]e)t]H}}}!?!3qo
};V](
CDk}?}?i*/int33
 19
class]stringnil<6
(){72
:=[)]int-)],	u0M9S0string352
9600
;>)[
T7
;(-,{9
2498
:=int:=]array}classOstring(extends -[array+; [;N9aIOclass<voidint- .77
stringzHmb;65
	"X"nil//
79
//(V;}!V;a
=.([{9795
(>-,{/*;*/(:={{;>class/*zK;!I}8V(J!
)
)];2P!?!V!}*/;.//3
. }classintarray]class]]=6
4
[>.:=]Z5e35
-2655
]=	,{array//Sc!)string=5470
P11R-{"F"string
int+
)nil>;45
+)>,{+34
""int=voidclass-)//K)2!(;!!w)1?]}];4
=.,<..1218
.string[1873
string/*)]}4!(}
?v]0}
*/{{483
>}<"g"8
intstringvoidextends,="p"[]=}()]53
60
"xE"""([86
4
nil+<classextends7533
y2Mw43TG"qeJ"class=nilvoidiFbg:=;->;:=	
int/*}
!}*/.(.,-void
71
"R"voidextends;8668
/*J?];0]!]j]?*/)""//g!?I
{+-.>}stringstring.30
()int<=
""<555
293
{:=+//]6>5304
void:=;extendsvoidarray//}?(}void>=;classvoid),). }643
3938
extends58
--):=72

.)(}nilvoid[>
nil5360
)/*
!F;?9!}7!*/,:=string/**/}"RYv":=,[string	58
831
string/*83?
9l
}]?;!;}!O}
6}?!?3]
;};];
moP]?;

)
665}*/5401
voidNj8Lh010
53
>nil/*]!A!9]n;
)]];kc;5}?

(]?]
]!!*/.=,}{1143
string 
aqiextends+-classnilnil>,//1}nil.class9
string+){	.nil+403
P	[int[,{[].array-=9
string(/*?!b!;;92;V(]}/;?)!])
*/string[),stringvoid+).) //array""3
string{33
<void;(	]+nil/*;(}*/}void]-+/*?}
;!/y*/-14
A],:=voidint+8456
//)y
;void	[void82
)//S
[+=)nil:=class:=;.}:=577
}stringvoid)[(/*}h*/:=
 extends+),{arraynilBnilint"NVPmJwqWMhhpC" int(-string}string:=