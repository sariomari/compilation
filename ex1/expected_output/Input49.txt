RBRACK[1,1]
ID(LFRquXbFnilarray)[1,3]
LBRACK[1,19]
INT(81)[1,20]
ASSIGN[2,1]
EXTENDS[2,3]
MINUS[2,10]
ARRAY[2,15]
RBRACE[2,20]
INT(8460)[2,21]
LBRACE[3,1]
SEMICOLON[3,2]
RBRACK[4,1]
INT(16)[4,2]
EQ[5,1]
SEMICOLON[6,1]
INT(600)[6,2]
STRING("")[7,1]
DOT[7,3]
ID(array62)[7,4]
MINUS[8,1]
ID(classint)[8,2]
LPAREN[8,10]
LT[8,11]
INT(48)[8,12]
LBRACE[9,1]
COMMA[9,2]
MINUS[9,3]
ID(carrayvoid)[9,4]
LPAREN[9,14]
LBRACE[9,15]
TYPE_STRING[9,16]
SEMICOLON[9,22]
INT(517)[9,23]
TYPE_VOID[10,1]
RBRACE[10,5]
LT[10,16]
LPAREN[10,17]
STRING("l")[10,18]
RBRACK[10,21]
CLASS[10,22]
STRING("zGSzAbLffRC")[10,27]
INT(2749)[10,40]
INT(310)[11,1]
LPAREN[12,1]
ID(classvoidvoidvoid)[12,2]
LPAREN[12,19]
COMMA[12,20]
PLUS[12,21]
PLUS[12,22]
GT[12,23]
ASSIGN[12,24]
ID(classz)[12,26]
COMMA[12,32]
COMMA[12,33]
GT[13,1]
RPAREN[13,2]
PLUS[13,3]
ARRAY[13,4]
RBRACE[13,9]
ID(string33)[13,10]
COMMA[14,1]
INT(40)[15,1]
LPAREN[16,1]
LBRACE[20,5]
INT(4000)[24,3]
LPAREN[25,1]
ID(arrayvoid)[25,2]
INT(500)[26,1]
INT(6)[27,1]
RBRACE[28,1]
COMMA[28,2]
ID(stringclass9837)[28,3]
NIL[29,1]
LBRACK[29,4]
RBRACE[29,5]
GT[29,6]
STRING("")[29,7]
INT(6562)[29,9]
TYPE_STRING[30,1]
LBRACE[30,7]
INT(5)[31,1]
ASSIGN[32,1]
ID(voidnil)[32,3]
LBRACK[32,10]
LBRACE[32,11]
ID(Cf)[32,12]
COMMA[32,14]
INT(9539)[32,15]
INT(7888)[33,1]
PLUS[34,1]
LBRACE[34,2]
EQ[34,3]
ID(Ad)[35,1]
RBRACE[35,3]
LBRACE[35,4]
INT(8)[35,5]
INT(24)[36,1]
EXTENDS[37,1]
PLUS[37,8]
RBRACK[37,9]
INT(910)[37,10]
INT(777)[38,1]
ASSIGN[39,1]
CLASS[39,3]
DOT[39,8]
TYPE_VOID[39,9]
DOT[39,13]
SEMICOLON[39,14]
ASSIGN[39,15]
RBRACK[39,17]
LT[40,1]
LBRACE[41,1]
TYPE_VOID[42,1]
RBRACE[42,5]
MINUS[42,6]
SEMICOLON[42,7]
STRING("f")[43,1]
RPAREN[43,4]
LPAREN[43,5]
ID(d)[43,6]
ASSIGN[43,7]
TYPE_STRING[43,9]
RBRACE[43,15]
INT(52)[43,16]
MINUS[44,1]
LBRACE[44,2]
EXTENDS[44,3]
EQ[44,10]
EXTENDS[44,11]
ID(a2lEextendsarray)[45,1]
LT[45,17]
STRING("zNb")[45,18]
ID(nil5699)[45,23]
LPAREN[47,1]
INT(5886)[47,2]
EQ[48,1]
LT[48,2]
PLUS[48,3]
RBRACK[48,4]
SEMICOLON[48,5]
CLASS[48,6]
RPAREN[48,11]
INT(39)[48,12]
COMMA[49,1]
INT(9)[49,2]
TYPE_INT[50,1]
LT[51,1]
EXTENDS[51,2]
MINUS[51,9]
LBRACE[51,10]
LT[51,11]
EXTENDS[51,12]
RBRACE[51,19]
SEMICOLON[51,20]
INT(7671)[51,21]
RPAREN[52,1]
INT(0)[52,2]
RBRACK[53,1]
INT(3)[53,2]
INT(178)[54,1]
EQ[55,1]
INT(41)[55,2]
LBRACE[56,1]
COMMA[56,2]
TYPE_STRING[56,3]
RPAREN[56,9]
INT(57)[56,10]
ID(classarray)[57,1]
LPAREN[57,11]
GT[57,12]
TYPE_STRING[57,13]
PLUS[57,19]
MINUS[57,20]
TYPE_VOID[57,21]
EQ[57,25]
INT(0)[57,26]
RBRACK[59,1]
RPAREN[59,2]
ID(extendsvoid)[59,3]
COMMA[59,14]
INT(8020)[59,15]
LPAREN[60,1]
TYPE_INT[60,2]
DOT[60,5]
LT[60,6]
ID(arraystring)[60,7]
LT[60,18]
LBRACK[60,19]
ASSIGN[60,20]
MINUS[60,22]
ID(stringvoid721)[61,12]
TYPE_VOID[62,1]
RPAREN[62,5]
MINUS[62,6]
NIL[62,7]
RBRACE[62,10]
ID(nilvoid)[62,11]
RBRACK[62,18]
PLUS[62,19]
LBRACE[62,20]
ID(M62)[62,21]
GT[62,24]
TYPE_VOID[62,25]
RBRACK[62,29]
ID(intvoidint)[62,30]
LT[62,40]
LT[62,41]
INT(9722)[62,42]
ID(xB)[63,1]
PLUS[63,3]
LPAREN[63,4]
COMMA[63,5]
TYPE_VOID[63,6]
RPAREN[63,10]
RBRACK[63,11]
CLASS[64,1]
LT[64,6]
INT(4047)[64,7]
SEMICOLON[65,1]
RBRACK[65,2]
RBRACE[65,3]
ID(intarray)[65,4]
TYPE_STRING[66,1]
ASSIGN[67,1]
DOT[67,3]
RBRACE[67,4]
DOT[67,5]
RBRACK[67,6]
MINUS[67,7]
GT[68,1]
INT(0)[68,2]
INT(87)[69,1]
INT(68)[70,1]
INT(5)[72,1]
ID(extendsint)[76,9]
LT[77,11]
SEMICOLON[77,12]
NIL[77,13]
INT(5)[77,29]
ID(yR7sz40x2ETc)[78,1]
NIL[81,10]
EQ[81,13]
INT(6)[81,14]
ID(jDKVQBBtclass)[82,1]
MINUS[82,14]
ID(intstringstring)[82,15]
LT[82,30]
ID(rcTextends)[82,31]
EQ[82,41]
NIL[82,42]
LPAREN[82,45]
INT(279)[83,5]
NIL[84,1]
RBRACE[84,4]
EQ[84,5]
STRING("")[84,6]
DOT[84,8]
LBRACE[84,10]
CLASS[84,11]
MINUS[84,16]
NIL[84,18]
STRING("")[84,21]
LT[84,23]
COMMA[84,24]
PLUS[84,25]
EXTENDS[84,26]
SEMICOLON[84,33]
ID(stringvoid)[84,34]
LT[84,44]
INT(5209)[84,45]
ID(U)[85,1]
LPAREN[85,2]
INT(781)[85,3]
INT(914)[86,1]
ARRAY[87,1]
COMMA[87,6]
INT(4)[88,2]
SEMICOLON[89,1]
CLASS[89,2]
PLUS[90,1]
CLASS[90,2]
PLUS[90,7]
CLASS[90,8]
PLUS[91,1]
LBRACK[91,2]
RBRACK[91,3]
TYPE_INT[91,4]
LBRACE[91,7]
ASSIGN[91,8]
LT[91,10]
MINUS[91,11]
LBRACK[91,12]
RBRACE[91,13]
INT(58)[91,14]
GT[92,1]
LBRACK[92,2]
GT[92,3]
RBRACK[93,1]
LBRACE[93,2]
RPAREN[93,3]
TYPE_STRING[93,4]
TYPE_INT[95,3]
SEMICOLON[95,6]
LPAREN[95,7]
RPAREN[95,8]
ID(intstring)[96,1]
COMMA[96,10]
TYPE_VOID[96,11]
LBRACK[96,15]
EQ[96,17]
TYPE_STRING[96,19]
EQ[96,25]
EQ[96,26]
LBRACE[96,27]
RBRACK[96,28]
ID(intlr)[96,29]
ASSIGN[96,34]
INT(73)[96,36]
EXTENDS[97,1]
EQ[97,8]
COMMA[97,9]
TYPE_INT[97,10]
CLASS[97,14]
LBRACE[97,19]
ID(class78)[97,20]
LBRACK[98,3]
CLASS[98,4]
STRING("A")[98,9]
INT(84)[98,21]
LPAREN[99,1]
RBRACK[99,2]
DOT[99,3]
EQ[102,4]
INT(3)[102,5]
PLUS[104,1]
NIL[104,2]
PLUS[104,5]
TYPE_INT[104,6]
ASSIGN[105,1]
LBRACE[105,3]
ID(classextends)[105,4]
COMMA[105,16]
INT(1556)[105,17]
LPAREN[106,1]
LBRACE[106,2]
EXTENDS[106,3]
COMMA[106,10]
RPAREN[107,1]
SEMICOLON[107,2]
NIL[107,3]
LPAREN[107,6]
ID(classstring)[107,8]
EQ[107,19]
ID(hint)[107,20]
COMMA[107,24]
INT(83)[107,25]
INT(526)[108,1]
EQ[109,1]
RBRACK[109,2]
SEMICOLON[109,3]
ID(intint)[109,4]
GT[109,10]
COMMA[109,11]
ARRAY[109,12]
DOT[109,17]
GT[109,18]
LBRACK[109,19]
INT(9284)[109,20]
LBRACK[110,1]
LT[120,6]
EQ[120,7]
GT[120,8]
LBRACE[120,9]
EXTENDS[120,11]
LPAREN[120,18]
INT(4)[120,19]
LT[121,1]
ASSIGN[121,2]
ID(arrayextends5)[121,4]
ID(classy)[122,1]
CLASS[123,1]
RPAREN[123,6]
INT(711)[123,7]
RBRACE[124,1]
EXTENDS[124,2]
CLASS[124,14]
EQ[124,19]
EXTENDS[124,20]
ID(stringy1)[125,1]
LPAREN[126,3]
LPAREN[126,4]
LBRACK[126,5]
ID(int33)[134,11]
INT(19)[135,2]
CLASS[136,1]
RBRACK[136,6]
ID(stringnil)[136,7]
LT[136,16]
INT(6)[136,17]
LPAREN[137,1]
RPAREN[137,2]
LBRACE[137,3]
INT(72)[137,4]
ASSIGN[138,1]
LBRACK[138,3]
RPAREN[138,4]
RBRACK[138,5]
TYPE_INT[138,6]
MINUS[138,9]
RPAREN[138,10]
RBRACK[138,11]
COMMA[138,12]
ID(u0M9S0string352)[138,14]
INT(9600)[139,1]
SEMICOLON[140,1]
GT[140,2]
RPAREN[140,3]
LBRACK[140,4]
ID(T7)[141,1]
SEMICOLON[142,1]
LPAREN[142,2]
MINUS[142,3]
COMMA[142,4]
LBRACE[142,5]
INT(9)[142,6]
INT(2498)[143,1]
ASSIGN[144,1]
TYPE_INT[144,3]
ASSIGN[144,6]
RBRACK[144,8]
ARRAY[145,1]
RBRACE[145,6]
ID(classOstring)[145,7]
LPAREN[145,19]
EXTENDS[145,20]
MINUS[145,28]
LBRACK[145,29]
ARRAY[145,30]
PLUS[145,35]
SEMICOLON[145,36]
LBRACK[145,38]
SEMICOLON[145,39]
ID(N9aIOclass)[145,40]
LT[145,50]
ID(voidint)[145,51]
MINUS[145,58]
DOT[145,60]
INT(77)[145,61]
ID(stringzHmb)[146,1]
SEMICOLON[146,11]
INT(65)[146,12]
STRING("X")[147,2]
NIL[147,5]
INT(79)[148,1]
EQ[150,1]
DOT[150,2]
LPAREN[150,3]
LBRACK[150,4]
LBRACE[150,5]
INT(9795)[150,6]
LPAREN[151,1]
GT[151,2]
MINUS[151,3]
COMMA[151,4]
LBRACE[151,5]
LPAREN[151,11]
ASSIGN[151,12]
LBRACE[151,14]
LBRACE[151,15]
SEMICOLON[151,16]
GT[151,17]
CLASS[151,18]
SEMICOLON[157,4]
DOT[157,5]
INT(3)[158,1]
DOT[159,1]
RBRACE[159,3]
ID(classintarray)[159,4]
RBRACK[159,17]
CLASS[159,18]
RBRACK[159,23]
RBRACK[159,24]
EQ[159,25]
INT(6)[159,26]
INT(4)[160,1]
LBRACK[162,1]
GT[163,1]
DOT[163,2]
ASSIGN[163,3]
RBRACK[164,1]
ID(Z5e35)[164,2]
MINUS[165,1]
INT(2655)[165,2]
RBRACK[166,1]
EQ[166,2]
COMMA[166,4]
LBRACE[166,5]
ARRAY[166,6]
RPAREN[167,1]
TYPE_STRING[167,2]
EQ[167,8]
INT(5470)[167,9]
ID(P11R)[168,1]
MINUS[168,5]
LBRACE[168,6]
STRING("F")[168,7]
TYPE_STRING[168,10]
TYPE_INT[169,1]
PLUS[169,4]
RPAREN[170,1]
NIL[170,2]
GT[170,5]
SEMICOLON[170,6]
INT(45)[170,7]
PLUS[171,1]
RPAREN[171,2]
GT[171,3]
COMMA[171,4]
LBRACE[171,5]
PLUS[171,6]
INT(34)[171,7]
STRING("")[172,1]
TYPE_INT[172,3]
EQ[172,6]
ID(voidclass)[172,7]
MINUS[172,16]
RPAREN[172,17]
INT(4)[173,1]
EQ[174,1]
DOT[174,2]
COMMA[174,3]
LT[174,4]
DOT[174,5]
DOT[174,6]
INT(1218)[174,7]
DOT[175,1]
TYPE_STRING[175,2]
LBRACK[175,8]
INT(1873)[175,9]
TYPE_STRING[176,1]
LBRACE[178,3]
LBRACE[178,4]
INT(483)[178,5]
GT[179,1]
RBRACE[179,2]
LT[179,3]
STRING("g")[179,4]
INT(8)[179,7]
ID(intstringvoidextends)[180,1]
COMMA[180,21]
EQ[180,22]
STRING("p")[180,23]
LBRACK[180,26]
RBRACK[180,27]
EQ[180,28]
RBRACE[180,29]
LPAREN[180,30]
RPAREN[180,31]
RBRACK[180,32]
INT(53)[180,33]
INT(60)[181,1]
STRING("xE")[182,1]
STRING("")[182,5]
LPAREN[182,7]
LBRACK[183,1]
INT(86)[183,2]
INT(4)[184,1]
NIL[185,1]
PLUS[185,4]
LT[185,5]
ID(classextends7533)[186,1]
ID(y2Mw43TG)[187,1]
STRING("qeJ")[187,9]
CLASS[187,14]
EQ[188,1]
ID(nilvoidiFbg)[188,2]
ASSIGN[188,13]
SEMICOLON[188,15]
MINUS[188,16]
GT[188,17]
SEMICOLON[188,18]
ASSIGN[188,19]
TYPE_INT[189,1]
DOT[190,5]
LPAREN[190,6]
DOT[190,7]
COMMA[190,8]
MINUS[190,9]
TYPE_VOID[190,10]
INT(71)[191,1]
STRING("R")[192,1]
ID(voidextends)[192,4]
SEMICOLON[192,15]
INT(8668)[192,16]
RPAREN[193,16]
STRING("")[193,17]
LBRACE[194,1]
PLUS[194,2]
MINUS[194,3]
DOT[194,4]
GT[194,5]
RBRACE[194,6]
ID(stringstring)[194,7]
DOT[194,19]
INT(30)[194,20]
LPAREN[195,1]
RPAREN[195,2]
TYPE_INT[195,3]
LT[195,6]
EQ[195,7]
STRING("")[196,1]
LT[196,3]
INT(555)[196,4]
INT(293)[197,1]
LBRACE[198,1]
ASSIGN[198,2]
PLUS[198,4]
GT[199,1]
INT(5304)[199,2]
TYPE_VOID[200,1]
ASSIGN[200,5]
SEMICOLON[200,7]
ID(extendsvoidarray)[200,8]
RBRACE[201,1]
TYPE_VOID[201,2]
GT[201,6]
EQ[201,7]
SEMICOLON[201,8]
ID(classvoid)[201,9]
RPAREN[201,18]
COMMA[201,19]
RPAREN[201,20]
DOT[201,21]
RBRACE[201,23]
INT(643)[201,24]
INT(3938)[202,1]
ID(extends58)[203,1]
MINUS[204,1]
MINUS[204,2]
RPAREN[204,3]
ASSIGN[204,4]
INT(72)[204,6]
DOT[206,1]
RPAREN[206,2]
LPAREN[206,3]
RBRACE[206,4]
ID(nilvoid)[206,5]
LBRACK[206,12]
GT[206,13]
ID(nil5360)[207,1]
RPAREN[208,1]
COMMA[209,12]
ASSIGN[209,13]
TYPE_STRING[209,15]
RBRACE[209,25]
STRING("RYv")[209,26]
ASSIGN[209,31]
COMMA[209,33]
LBRACK[209,34]
TYPE_STRING[209,35]
INT(58)[209,42]
INT(831)[210,1]
TYPE_STRING[211,1]
INT(5401)[225,7]
ID(voidNj8Lh010)[226,1]
INT(53)[227,1]
GT[228,1]
NIL[228,2]
DOT[234,6]
EQ[234,7]
COMMA[234,8]
RBRACE[234,9]
LBRACE[234,10]
INT(1143)[234,11]
TYPE_STRING[235,1]
ID(aqiextends)[236,1]
PLUS[236,11]
MINUS[236,12]
ID(classnilnil)[236,13]
GT[236,24]
COMMA[236,25]
NIL[237,1]
DOT[237,4]
ID(class9)[237,5]
TYPE_STRING[238,1]
PLUS[238,7]
RPAREN[238,8]
LBRACE[238,9]
DOT[238,11]
NIL[238,12]
PLUS[238,15]
INT(403)[238,16]
ID(P)[239,1]
LBRACK[239,3]
TYPE_INT[239,4]
LBRACK[239,7]
COMMA[239,8]
LBRACE[239,9]
LBRACK[239,10]
RBRACK[239,11]
DOT[239,12]
ARRAY[239,13]
MINUS[239,18]
EQ[239,19]
INT(9)[239,20]
TYPE_STRING[240,1]
LPAREN[240,7]
TYPE_STRING[244,3]
LBRACK[244,9]
RPAREN[244,10]
COMMA[244,11]
ID(stringvoid)[244,12]
PLUS[244,22]
RPAREN[244,23]
DOT[244,24]
RPAREN[244,25]
ARRAY[245,1]
STRING("")[245,6]
INT(3)[245,8]
TYPE_STRING[246,1]
LBRACE[246,7]
INT(33)[246,8]
LT[247,1]
TYPE_VOID[247,2]
SEMICOLON[247,6]
LPAREN[247,7]
RBRACK[247,9]
PLUS[247,10]
NIL[247,11]
RBRACE[248,4]
TYPE_VOID[248,5]
RBRACK[248,9]
MINUS[248,10]
PLUS[248,11]
MINUS[249,7]
INT(14)[249,8]
ID(A)[250,1]
RBRACK[250,2]
COMMA[250,3]
ASSIGN[250,4]
ID(voidint)[250,6]
PLUS[250,13]
INT(8456)[250,14]
SEMICOLON[253,1]
TYPE_VOID[253,2]
LBRACK[253,7]
ID(void82)[253,8]
RPAREN[254,1]
LBRACK[255,1]
PLUS[255,2]
EQ[255,3]
RPAREN[255,4]
NIL[255,5]
ASSIGN[255,8]
CLASS[255,10]
ASSIGN[255,15]
SEMICOLON[255,17]
DOT[255,18]
RBRACE[255,19]
ASSIGN[255,20]
INT(577)[255,22]
RBRACE[256,1]
ID(stringvoid)[256,2]
RPAREN[256,12]
LBRACK[256,13]
LPAREN[256,14]
ASSIGN[256,21]
EXTENDS[257,2]
PLUS[257,9]
RPAREN[257,10]
COMMA[257,11]
LBRACE[257,12]
ID(arraynilBnilint)[257,13]
STRING("NVPmJwqWMhhpC")[257,28]
TYPE_INT[257,44]
LPAREN[257,47]
MINUS[258,1]
TYPE_STRING[258,2]
RBRACE[258,8]
TYPE_STRING[258,9]
ASSIGN[259,1]